<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoAPI Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="read_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="write_lineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="read_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный файл:</string>
    </property>
   </widget>
   <widget class="QLabel" name="write_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Конечный файл:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль (опционально):</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="enc_dec_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Шифрование / Дешифрование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encrypt_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Шифровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Дешифровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sign_ver_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Подписывание сообщения / Верификация сообщения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подписать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verify_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Верифицировать</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_file1_pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_file2_pushButton_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_cert_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сертификат:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_cert_pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choose_cert_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Подписываемое / верифицируемое сообщение:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="message_pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="sign_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл с подписью:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sign_lineEdit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>430</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="display_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Подпись:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
